src/main/resources/lib/auth/blockWrapper.jelly

```diff
@@ -0,0 +1,16 @@
+<?jelly escape-by-default='true'?>
```

Comment:
you're missing the empty marker file 'taglib'

Reply:
It's already there on the repository. see https://github.com/jenkinsci/google-oauth-plugin/pull/99

Reply:
ah sure, I'm used to people missing that and that being the reason why the build has failed